### DISCUSSION TITLE
Weird error while using Nativewind and Expo Router

### DISCUSSION THREAD

**Discussion by rafaelrcamargo** [expo/router#219] [Q&A]
I'm going through this error while trying to recreate the tutorial using the navigation, but using that with the Nativewind package, do you happen to have any clues?

>  ERROR  The action 'GO_BACK' was not handled by any navigator.

![image](https://user-images.githubusercontent.com/66796237/213943937-ba7489c9-881c-4b1f-b1ba-39ebce469081.png)

## Layout:

```ts
import { Stack } from "expo-router"

export default function Layout() {
  return (
    <Stack
      initialRouteName="home"
      screenOptions={{
        headerStyle: {
          backgroundColor: "#f4511e"
        },
        headerTintColor: "#fff",
        headerTitleStyle: {
          fontWeight: "bold"
        }
      }}
    />
  )
}
```

## Home:

```ts
import { View } from "react-native"
import { Link, Stack } from "expo-router"

export default function Home() {
  return (
    <View style={{ flex: 1, justifyContent: "center", alignItems: "center" }}>
      {/* Use the `Screen` component to configure the layout. */}
      <Stack.Screen options={{ title: "Overview" }} />
      {/* Use the `Link` component to enable optimized client-side routing. */}
      <Link href="/details">Go to Details</Link>
    </View>
  )
}
```

> Ps: No route is found, and if I go through the Sitemap, no navigator is loaded just the page content.

**Comment by firasrg** [comment]
Hello!

If Home screen component is declared inside index.tsx, then change initialRouteName to "index".

The ``Stack`` JSX must be defined in _layout.tsx file like the following: 

```tsx
return(
  <Stack>
     <Stack.Screen name="details" />
  </Stack>
)
```

You don't have to add a stack for home if its inside index.tsx, because this is done automatically.

I hope it helps.
